android/src/main/java/com/tailscale/ipn/ui/view/ManagedByView.kt

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2024 Tailscale Inc & AUTHORS All rights reserved.
```

Comment:
Wrong header... James update it.  You can set a default in Android Studio for new files.